runtime/protocol/http-client/common/src/aws/smithy/kotlin/runtime/http/operation/HttpOperationContext.kt

```diff
@@ -50,6 +51,11 @@ public object HttpOperationContext {
      * Cached attribute level attributes (e.g. rpc.method, rpc.service, etc)
      */
     public val OperationAttributes: AttributeKey<Attributes> = AttributeKey("aws.smithy.kotlin#OperationAttributes")
+
+    /**
+     * The clock skew duration to apply to any time-based calculations in the operation (e.g. signing)
+     */
+    public val ClockSkew: AttributeKey<Duration> = AttributeKey("aws.smithy.kotlin#ClockSkew")
```

Comment:
**Correctness**: The KDocs say "any time-based calculations in the operation" but signature calculation is _the only_ place this is used. It's not applied to retry delays, deserialization, etc.

Reply:
Thanks for catching that. I wrote this description before doing the implementation, when I still had planned to update **all** time-based calculations in the SDK, but I think just signature calculation is enough for now.